docs/templating/twig_function_reference/product_twig_functions.md

```diff
@@ -129,4 +129,15 @@ The `ibexa_is_pim_local` is a helper Twig function that enables changing the beh
         <button type="button">Modify product data</button>
     </div>
 {% endif %}
+```
+
+### `ibexa_product_catalog_group_attributes`
+
+The `ibexa_product_catalog_group_attributes` filter groups product attributes (configured in `ibexa_product_catalog_group_attributes`).
```

Comment:
```suggestion
The `ibexa_product_catalog_group_attributes` filter groups product attributes based on the [attribute group](https://doc.ibexa.co/projects/userguide/en/latest/pim/work_with_product_attributes/) they belong to. .
```

I'd mention what is used for the grouping - the attribute group. For the link: I didn't find a better mention of attribute groups in the doc. If you decide to use it, please use the `[[= user_doc ==]` variable.

> configured in `ibexa_product_catalog_group_attributes`

I don't understand this part, I don't think there' any configuration?